config_management/group_vars/all

```diff
@@ -2,7 +2,7 @@
 go_version: 1.7.4
 terraform_version: 0.8.5
 docker_version: 1.11.2
-kubernetes_version: 1.5.2
+kubernetes_version: 1.6.0-beta.2
```

Comment:
> Do you think setting an "unstable" (beta/alpha) version as a default could make our own CIT more unstable?

Yes, most likely. I just think it'd be good to have a simple way of running testing agains alphas and betas next time.